ui/push-health/UnsupportedGroup.jsx

```diff
@@ -0,0 +1,94 @@
+/* eslint-disable jsx-a11y/no-noninteractive-element-interactions */
```

Comment:
I'm curious why you disabled this rule instead of getting it to pass, since it's new code.

Reply:
This was cut and pasted from another component.  So I just didn't think about it.  Fixed.